docs/installation/metadata-types.rst

```diff
@@ -14,8 +14,8 @@ To add or alter metadata types, you can use commands like: ``datacube metadata a
 and to update: ``datacube metadata update <path-to-file>``. Using ``--allow-unsafe`` will allow
 you to update metadata types where the changes may have unexpected consequences.
 
-Note that the postgis driver only supports eo3-compatible metadata types, and from version 2.0 onward, support for non-eo3-compatible metadata types
-will be fully deprecated.
+Note that the postgis driver only supports `eo3-compatible metadata types`_, and from version 2.0 onward,
```

Comment:
I find the sentence hard to read. Not sure how to fix it, but perhaps "and from version .." => "and support for non-eo3-compatible metadata types will be deprecated in version 2.0." makes it slightly easier to read?